package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "glslify-loader",
-  "version": "1.0.2",
-  "description": "glslify loader module for webpack",
+  "version": "1.0.3",
```

Comment:
This seems like more than a patch change. Perhaps even a major version. Best to leave the version untouched in a PR and then increment as needed when publishing.

Reply:
I can do that 👍

Reply:
I wouldn't say a major though, cause no breaking changes were introduced.

Reply:
hmmm... I guess? or is the webpack update considered one?

Reply:
Is webpack compatibility changed? My concern is that someone with a working webpack setup will find that this plugin no longer works.